infoschema/slow_log_test.go

```diff
@@ -123,7 +123,7 @@ select * from t;
 	reader = bufio.NewReader(slowLog)
 	_, err = infoschema.ParseSlowLog(loc, reader)
 	c.Assert(err, NotNil)
-	c.Assert(err.Error(), Equals, "parse slow log failed `Succ` error: strconv.ParseBool: parsing \"abc\": invalid syntax")
+	c.Assert(err.Error(), Equals, "parse slow log at line 2 failed. Field: `Succ`, error: strconv.ParseBool: parsing \"abc\": invalid syntax")
```

Comment:
parse => Parse.
error: strconv.ParseBool: parsing \"abc\": invalid syntax => value: \"abc\"

Reply:
The error message is the original error message from `strconv.ParseBool`, I think to print it out is better.

Reply:
@djshow832 you can try suggestion mode, which will be more clear.